File: src/ast/NoexceptSpec.java
package ast;

import java.io.StringWriter;

public class NoexceptSpec extends ParentNode {
    public NoexceptSpec(BaseNode child) {
        super(NodeType.NoexceptSpec, child);
    }

    @Override
    public void PrintLeft(StringWriter writer) {
        writer.write("noexcept(");
        child.Print(writer);
        writer.write(")");
    }
}

File: src/de/fearlesstobi/demangler/ast/NoexceptSpec.kt
package de.fearlesstobi.demangler.ast

import java.io.StringWriter

class NoexceptSpec(child: BaseNode?) : ParentNode(NodeType.NoexceptSpec, child) {
    public override fun printLeft(writer: StringWriter) {
        writer.write("noexcept(")
        child!!.print(writer)
        writer.write(")")
    }
}